portfolio/src/main/webapp/WEB-INF/city-data.csv

```diff
@@ -0,0 +1,12 @@
+Jakarta,-6.175540,106.82743,2000 - Present
+London,51.514248,-0.09314520,2018 - Present
+Florence,43.766667,11.25,June 2019
+Budapest,47.5,19.083333,April 2019
+Santorini,36.3932,25.4615,July 2019
+Seoul,37.5985,126.9783,December 2019
+Prague,50.0755,14.4378,March 2019
+Bali,-8.3405,115.0920,May 2018
+Paris,48.8566,2.3522,September 2018
+Surabaya,-7.2575,112.7521,June 2020
+Bandung,-6.9175,107.6191,June 2020
+Edinburgh,55.9533,-3.1883,February 2020
```

Comment:
I don't think this file should be in `WEB-INF`, I'd move it to the server code directory as it's only used there under a directory called `resources` maybe?

From https://docs.oracle.com/cd/E13222_01/wls/docs90/webapp/configurewebapp.html
> The WEB-INF directory contains the deployment descriptors for the Web application (web.xml and weblogic.xml) and two subdirectories for storing compiled Java classes and library JAR files. These subdirectories are respectively named classes and lib. JSP taglibs are stored in the WEB-INF directory at the top level of the staging directory. The Java classes include servlets, helper classes and, if desired, precompiled JSPs.

Reply:
Thanks Mostafa for the feedback! May I know that the server code directory is, is it under "main" or "webapp"?
I was following the directory structure from the example
![Screenshot 2020-07-30 at 8 18 54 AM](https://user-images.githubusercontent.com/41379082/88893013-a158a280-d23d-11ea-9638-2ac18f59dbbf.png)

Reply:
Thanks Rania! I don't know why it's added there in the example but looks like this is not the best practice :)
The server code is the one under `portfolio/src/main/java/com/google/sps/` (i.e. where you handle the requests and where you also read the file), anything under `portfolio/src/main/webapp/` is called the client code (i.e. where you call the server and have the UI components)

Reply:
Got it, thank you Mostafa, will make the change now :D